tests/impl.cpp

```diff
@@ -11,7 +11,40 @@
 // This program a set of unit tests to ensure that each SSE call provide the
 // output we expect.  If this fires an assert, then something didn't match up.
 
+#ifdef __aarch64__
```

Comment:
We should take both Arm and Aarch64 into considerations.

Reply:
Good point.  I will update this patch.  Thanks for your review!

Reply:
I tried to enable arm32 builds with TravisCI as well, however arm32 is not yet supported:
https://docs.travis-ci.com/user/multi-cpu-architectures
https://travis-ci.community/t/how-to-deploy-project-with-multiple-architectures-how-well-does-arm32-work/7679